narwhals/expr.py

```diff
@@ -936,6 +939,9 @@ def cum_sum(self: Self, *, reverse: bool = False) -> Self:
     def diff(self: Self) -> Self:
         """Returns the difference between each element and the previous one.
 
+        For lazy backends, this operation must be followed by `Expr.over` with `order_by`
```

Comment:
Wrapping in a `!!! info` or even `!!! warning` [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) might be useful

Reply:
sure, thanks!